tests/integration/components/frost-bunsen-detail/selected-tab-name-test.js

```diff
@@ -0,0 +1,61 @@
+import {expect} from 'chai'
```

Comment:
I've added new test helpers I'd like to see you use here as well as a test to verify you can update the `selectedTabLabel` property to programmatically change the tab.

```js
import {expect} from 'chai'
import {setupFormComponentTest} from 'dummy/tests/helpers/utils'
import hbs from 'htmlbars-inline-precompile'
import {beforeEach, describe, it} from 'mocha'

describe('Integration: Component / frost-bunsen-detail / open specific tab using tab label', function () {
  setupFormComponentTest({
    bunsenModel: {
      properties: {
        bar: {type: 'number'},
        baz: {type: 'boolean'},
        foo: {type: 'string'}
      },
      type: 'object'
    },
    bunsenView: {
      cellDefinitions: {
        one: {
          children: [
             {model: 'foo'},
             {model: 'bar'}
          ]
        },
        two: {
          children: [
             {model: 'baz'}
          ]
        }
      },
      cells: [
         {label: 'One', extends: 'one'},
         {label: 'Two', extends: 'two'}
      ],
      type: 'form',
      version: '2.0'
    },
    selectedTabLabel: 'Two'
  })

  it('renders second tab', function () {
    expect(this.$('.frost-tabs .active div').text().trim()).to.equal('Two')
  })

  describe('when selectedTab property updated to be first tab', function () {
    beforeEach(function () {
      this.set('selectedTabLabel', 'One')
    })

    it('renders first tab', function () {
      expect(this.$('.frost-tabs .active div').text().trim()).to.equal('One')
    })
  })
})
```